app/react/browser/createRoutes.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { IndexRoute, Route } from 'react-router';
+
+import App from './app/App.react';
+import Homepage from './homepage/Page.react';
+import Project from './projects/Page.react';
+
+export default function createRoutes() {
```

Comment:
Why? With readax-router-redux it's better to handle Google Analytics in middleware.